specification/maps/data-plane/DataRegistry/stable/2022-12-01/dataregistry.json

```diff
@@ -0,0 +1,716 @@
+        "operationId": "DataRegistries_List",
+        "x-ms-examples": {
+          "List all the previously uploaded data": {
+            "$ref": "./examples/DataRegistries_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List data request completed successfully. The response body contains a list of metadata of all the previously registered data registries.",
+            "schema": {
+              "$ref": "#/definitions/MetadataListResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/mapData/{udid}": {
+      "patch": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe endpoint of DataRegistry APIs is bound to the region of your maps account. Global maps account is not be able to use this service.\n\nThe DataRegistry Patch API allows the caller to patch a previously registered data registry.\n\nYou can use this API in a scenario like adding or removing geofences to or from an existing collection of geofences. Geofences are registered using the [DataRegistry Register API](https://docs.microsoft.com/rest/api/maps/data-registry/Register), for use in the [Azure Maps Geofencing Service](https://docs.microsoft.com/rest/api/maps/spatial).\n\nThis API is a synchronous API which may eventually fail and revert even if it returns `200 OK`. Please check [DataRegistry GetMetadata API](https://docs.microsoft.com/en-us/rest/api/maps/data-registry/get-metadata) for the processing status.\n\n## Submit Patch Request\n\nTo patch a data registry you will use a `PATCH` request. The request body will contain an object representing a patch to the data registry, in [JSON Merge Patch (RFC7396)](https://datatracker.ietf.org/doc/html/rfc7396). The path will contain the `udid` of the data registry to be update.\n\nFor example, to just change the connection object of a data registry that were previously registered without changing other properties of the data registry, set the `udid` in the query parameter and the new connection object in the body.",
+        "operationId": "DataRegistries_Patch",
```

Comment:
Better to use "Update" rather than "Patch" in the operationId.

Reply:
Fixed